test_task_packed_picking/tests/test_create_picking_and_put_product_a_package.py

```diff
@@ -0,0 +1,137 @@
+from odoo.tests import common, tagged
+
+
+@tagged("nice")
+class TestCreatePickingAndPutProductPackage(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.wizard = self.env["test.task.packed.picking.wizard"]
+        self.stock_picking = self.env["stock.picking"]
+        self.stock_move = self.env["stock.move"]
+        self.stock_location = self.env["stock.location"]
+        self.stock_picking_type = self.env["stock.picking.type"]
+        self.stock_picking_type_pack = self.env.ref("stock.picking_type_internal")
+        self.stock_picking_type_pack.write(
+            {
+                "code": "internal",
+                "name": "Internal",
+                "sequence_code": "INT",
+                "default_location_src_id": self.stock_location.create(
+                    {"name": "Stock", "usage": "internal"}
+                ).id,
+                "default_location_dest_id": self.stock_location.create(
+                    {"name": "Customers", "usage": "customer"}
+                ).id,
+            }
+        )
+        self.product = self.env["product.product"].create(
+            {"name": "Product", "type": "product"}
+        )
+        self.product2 = self.env["product.product"].create(
+            {"name": "Product2", "type": "product"}
+        )
+        self.product3 = self.env["product.product"].create(
+            {"name": "Product3", "type": "product"}
+        )
+        self.line_ids = [
+            (
+                0,
+                0,
+                {
+                    "product_id": self.product.id,
+                    "qty_done": 1,
+                },
+            ),
+            (
+                0,
+                0,
+                {
+                    "product_id": self.product2.id,
+                    "qty_done": 1,
+                },
+            ),
+            (
+                0,
+                0,
+                {
+                    "product_id": self.product3.id,
+                    "qty_done": 1,
+                    "serial": "1234567890",
+                },
+            ),
+        ]
+
+    def test_create_picking_and_put_product_a_package(self):
+        wizard = self.wizard.create(
```

Comment:
When testing wizards or any other form related logic better use the "Form" for that. Here is some [documentation](https://www.odoo.com/documentation/16.0/developer/reference/backend/testing.html#odoo.tests.common.Form)